src/ArmTemplates/Properties/launchSettings.json

```diff
@@ -1,7 +1,8 @@
 {
   "profiles": {
     "apimtemplate": {
-      "commandName": "Project"
+      "commandName": "Project",
+      "commandLineArgs": "extract --extractorConfig C:\\Projects\\other\\extractorConfig.json"
```

Comment:
Is this an example or for your testing? If so, you might want to move this out?

Reply:
deleted this. I have added ignore of changes this file to .gitignore, but somehow it was still pushed :(